runtime/doc/eval.txt

```diff
@@ -5303,7 +5314,6 @@ reltimestr({time})				*reltimestr()*
 		can use split() to remove it. >
 			echo split(reltimestr(reltime(start)))[0]
 <		Also see |profiling|.
-		{only available when compiled with the |+reltime| feature}
```

Comment:
The note `The accuracy depends on the system.` can be removed also (line 5301/5312). Nvim's `reltimestr()` should be very accurate on all supported systems. (We removed use of `gettimeofday()` in db7cd61f6297aab6343f7c6cd424d8987ca03a58)